poller/poller.go

```diff
@@ -34,6 +42,20 @@ type SplitData struct {
 	UsingSegmentsCount int
 }
 
+// Cache contains raw split data as well as the data in serialized format
+type Cache struct {
+	SplitData
+	SerializedData string
+}
+
+// SplitCachePreload contains the split/segment data from split.io
```

Comment:
nit: can we possibly note here that this struct has the same structure as SplitData but Splits and Segments are strings format